model/clusters_mgmt/v1/log_forwarder_group_versions_type.model

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Represents a log forwarder group versions configuration.
+struct LogForwarderGroupVersions {
+    // The name of the log forwarder group.
+    Name String
+
+    // Indicates whether this group is enabled and available for use.
+    Enabled Boolean
```

Comment:
@willkutler In general we've tried to move away from boolean indicators of enabled/disabled see: https://gitlab.cee.redhat.com/service/api-guidelines#avoid-boolean-fields-and-binary-state-wording

If enabling/disabling groups is something you intend to support in the current scope of implementation, then I would suggest re-naming this field to `state` as a string, with current values of `enabled` and `disabled`.